ansible/roles/test/tasks/decap.yml

```diff
@@ -50,27 +50,11 @@
 
 - set_fact:
     testname: decap
-    run_dir: /tmp
-    out_dir: /tmp/ansible-loganalyzer-results
-    test_match_file: decap_match_messages.txt
-    test_ignore_file: decap_ignore_messages.txt
-    test_expect_file: decap_expect_messages.txt
-    match_file: loganalyzer_common_match.txt
-    ignore_file: loganalyzer_common_ignore.txt
-    tests_location: "{{ 'roles/test/tasks' }}"
```

Comment:
We are not using loganalyzer in this test, so we don't have to use those files as well.